pyblhost/pyblhost.py

```diff
@@ -173,13 +173,15 @@ def __init__(self, logger: logging.Logger) -> None:
 
         # Flags used when uploading
         self._ack_response_event = threading.Event()
+        self._nack_response_event = threading.Event()
         self._reset_response_event = threading.Event()
         self._flash_erase_region_response_event = threading.Event()
         self._read_memory_response_tag_event = threading.Event()
         self._write_memory_response_event = threading.Event()
         self._read_memory_response_event = threading.Event()
         self._data_event = threading.Event()
         self._ping_response_event = threading.Event()
+        self._get_command_response_event = threading.Event()
```

Comment:
```suggestion
        self._get_property_response_event = threading.Event()
```

Reply:
I added a comment in the `get_property` method explaining the "nuance" of the old KBOOT. I hope that explains why I did it this way.

Reply:
Okay make sense then :)